pallets/pool-fees/src/lib.rs

```diff
@@ -492,11 +493,10 @@ pub mod pallet {
 				Error::<T>::PoolNotFound
 			);
 
-			let (_, _count) =
+			let (_, count) =
 				Self::update_portfolio_valuation_for_pool(pool_id, &mut T::Balance::zero())?;
 
-			// Ok(Some(T::WeightInfo::update_portfolio_valuation(count)).into())
-			Ok(Some(T::DbWeight::get().reads(1)).into())
+			Ok(Some(T::WeightInfo::update_portfolio_valuation(count)).into())
```

Comment:
The above weights attached to this extrinsic is 
```rust
#[pallet::weight(Weight::from_parts(10_000, 0))]
```

Should we change it to:

```rust
#[pallet::weight(Weight::update_porfolio_valuation(T::MaxFeesPeerPool::get()))]
```

to get a real maximum suggestion of the expected fees for this extrinsic?